dances.json

```diff
@@ -742,6 +742,13 @@
     "url": "https://www.commonfloorcontra.dance/",
     "weekdays": "Sundays"
   },
+  {
+    "annual_freq": 52,
+    "city": "Portland ME",
+    "gender_free": true,
+    "url": "https://www.portlandintowncontradance.com/",
```

Comment:
I guess this is ok, though I'd been thinking of waiting until they'd actually resumed?

Reply:
It's more that if someone searches today then we'll put them on site for a group that isn't actually hosting dances.

But delaying the PR seems like too much trouble, so I'm happy for this to go in. Probably won't affect anyone.